dc-image-processing/schedule.md

```diff
@@ -17,7 +17,7 @@ calendar, so you can take full advantage of the workshop.
     </table>
   </div>
   <div class="col-md-6">
-    <h3>Day 2. Thur 15 September 2023</h3>
+    <h3>Day 2. Thur 15 June 2023</h3>
```

Comment:
I assume you mean day 1? Also, since this is the generic metadata that is used for every workshop I suggest you leave out the date (even though it was there in the example)
```suggestion
    <h3>Day 1</h3>
```

Reply:
changed...plus more specific session names